src/App.js

```diff
@@ -1,45 +1,39 @@
-import React, { useState } from 'react';
+import React from 'react';
+import {
+  BrowserRouter as Router, Route, Link, Routes,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
 import HomePage from './components/HomePage';
 import FunnyQuot from './components/FunnyQuot';
 import './components/calculator.css';
 
```

Comment:
![image](https://github.com/jubaer919/math-magicians/assets/93039359/e13ba667-9cc6-42da-a4fd-ba91bdb77197)


- [ ] It would be nice if you remove the Live Demo section since you don't have any live link for your app 😨 Kindly remove the live demo section to make your readme descriptive and professional 👍